Dockerfile

```diff
@@ -1,16 +1,14 @@
-FROM golang:1.6-alpine
+FROM golang:1.7-alpine
 
 ENV PATH $PATH:/opt/flamegraph
 
 RUN apk --update add git && \
     apk add perl && \
-    curl -OL https://github.com/Masterminds/glide/releases/download/0.10.1/glide-0.10.1-linux-amd64.tar.gz && \
-    tar -xzf glide-0.10.1-linux-amd64.tar.gz && \
-    mv linux-amd64/glide /usr/bin && \
+    go get github.com/Masterminds/glide && \
```

Comment:
true, but only if you explicitly install curl:
https://gist.github.com/billf/13f69acf6ad4a8e8f9b8fdf745310fb9